tutorials/rust/harfbuzz/Cargo.toml

```diff
@@ -4,11 +4,9 @@
 
 [package]
 name = "tutorial_harfbuzz"
-publish = false
-license = "Unicode-3.0"
 edition = "2021"
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+publish = false
 
 [dependencies]
-icu_harfbuzz = { version = "0.1.2", features = ["compiled_data"] }
+icu_harfbuzz = { version = "0.6.0", features = ["compiled_data"] }
 harfbuzz = { version = "0.6.0" }
```

Comment:
note: I've set `icu_harfbuzz` to 0.6.0 to align with the used `harfbuzz` version